helper/chroot/chroot.go

```diff
@@ -182,10 +161,6 @@ func (fs *ChrootHelper) Lstat(filename string) (os.FileInfo, error) {
 }
 
 func (fs *ChrootHelper) Symlink(target, link string) error {
-	if !fs.symlinkSupport {
-		return billy.ErrNotSupported
-	}
-
 	target = filepath.FromSlash(target)
```

Comment:
All the path arguments at any implementation of fs support Linux-style paths, those are transformed to the specif one internally, this is well covered by the tests.